servicebox-app/src/main/resources/templates/serviceItem-old.html

```diff
@@ -0,0 +1,7 @@
+{@io.halkyon.model.Service service}
```

Comment:
Why do we have `file-old.html` ?

Reply:
oups, no, I forgot to remove them from the git index